recipe/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "titus2" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dd7ca43e2a7cde530e0ae440cb65d87aac9eedd8cdba9c47cff5d1882e88c297
+  patches:
+    - patches/0001-skip-test-for-darwin.patch  # [osx]
+
+build:
+  skip: True  # [py>=310]
+  # missing win build for avro --> avro-python3
+  skip: True  # [win]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  number: 0
+
+requirements:
+  build:
+    - patch  # [osx]
```

```suggestion

build:
  skip: True  # [py>=310]
  # missing win build for avro --> avro-python3
  skip: True  # [win]
  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
  number: 0

requirements:
```

Patch is unnecessary if using the pytest solution I mentioned.

---

Agreed. Thanks for the great suggestion!